### PR TITLE
ML-2704: Fix redis NaN conversion

### PR DESCRIPTION
**Pull request by alxtkr77**

### CODE REVIEW

File: storey/redis_driver.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def convert_redis_value_to_python_obj(cls, value):
             ret = math.inf
         elif value == "-inf":
             ret = -math.inf
-        elif value == "nan" or value == "-nan":
```
Comment (gtopper) [inline storey/redis_driver.py:197]: Any chance we will also get `"-0/0"`?

File: storey/redis_driver.py [hunk 2]
```diff
@@ -194,7 +194,7 @@ def convert_redis_value_to_python_obj(cls, value):
...
+        elif value == "nan" or value == "-nan" or value == "0/0":
```
Comment (gtopper) [inline storey/redis_driver.py]: Any chance we will also get "-0/0"?

Comment (alxtkr77) [reply storey/redis_driver.py]: Don't see how, but on the other hand Assaf added "-nan". I will add -0/0 as well to keep things symmetric.

**Review by gtopper — APPROVED** [review]
LGTM ✅

**Pull request merged**